### ISSUE TITLE
Error running example SPARQL query

### ISSUE THREAD

**Issue by aborruso** [maxlath/wikibase-cli#95]
Hi,
I'm testing the example SPARQL query

```
SELECT ?work WHERE {

  ?work wdt:P50 wd:Q42 .

}
```

and when I run `wb sparql test.rq` I have

```
/home/aborruso/n/lib/node_modules/wikibase-cli/node_modules/wikibase-sdk/lib/wikibase-sdk.js:62
    throw new Error(`invalid ${name}: ${url}`)
    ^

Error: invalid instance: undefined
    at validateEndpoint (/home/aborruso/n/lib/node_modules/wikibase-cli/node_modules/wikibase-sdk/lib/wikibase-sdk.js:62:11)
    at WBK (/home/aborruso/n/lib/node_modules/wikibase-cli/node_modules/wikibase-sdk/lib/wikibase-sdk.js:15:3)
    at module.exports (/home/aborruso/n/lib/node_modules/wikibase-cli/lib/wbk.js:6:10)
    at Object.<anonymous> (/home/aborruso/n/lib/node_modules/wikibase-cli/lib/make_sparql_query.js:7:56)
    at Module._compile (internal/modules/cjs/loader.js:774:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:785:10)
    at Module.load (internal/modules/cjs/loader.js:641:32)
    at Function.Module._load (internal/modules/cjs/loader.js:556:12)
    at Module.require (internal/modules/cjs/loader.js:681:19)
    at require (internal/modules/cjs/helpers.js:16:16)
    at Object.<anonymous> (/home/aborruso/n/lib/node_modules/wikibase-cli/bin/wb-sparql:12:25)
    at Module._compile (internal/modules/cjs/loader.js:774:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:785:10)
    at Module.load (internal/modules/cjs/loader.js:641:32)
    at Function.Module._load (internal/modules/cjs/loader.js:556:12)
    at Function.Module.runMain (internal/modules/cjs/loader.js:837:10)
```

I have set

    wb config sparql-endpoint https://query.wikidata.org/sparql

How to solve it?

Thank you

**Comment by maxlath** [comment]
[`wikibase-sdk` was throwing in absence of a Wikibase API `instance`](https://github.com/maxlath/wikibase-sdk/commit/92e4f95), this should be fixed in `wikibase-cli v8.1.2`

**Comment by aborruso** [comment]
@maxlath it works, thank you very much
